src/main/java/org/jabref/logic/importer/fileformat/CffImporter.java

```diff
@@ -0,0 +1,155 @@
+        CffFormat citation = mapper.readValue(reader, CffFormat.class);
+        HashMap<Field, String> entryMap = new HashMap<Field, String>();
+
+        HashMap<String, StandardField> fieldMap = getFieldMappings();
+        for (Map.Entry<String, String> property : citation.vals.entrySet()) {
+            if (fieldMap.containsKey(property.getKey())) {
+                entryMap.put(fieldMap.get(property.getKey()), property.getValue());
+            }
+        }
+
+        List<String> authors = new ArrayList<String>();
+        for (CffAuthor auth: citation.authors) {
+            String aName = auth.vals.get("name");
+            String aGivenNames = auth.vals.get("given-names");
+            String aFamilyNames = auth.vals.get("family-names");
+            String aNameParticle = auth.vals.get("name-particle");
+            String aAlias = auth.vals.get("alias");
+
+            if (aName != null) {
+                authors.add(aName);
+            } else if (aFamilyNames != null && aNameParticle != null && aGivenNames != null) {
+                authors.add(aGivenNames + " " + aNameParticle + " " + aFamilyNames);
+            } else if (aFamilyNames != null && aGivenNames != null) {
+                authors.add(aGivenNames + " " + aFamilyNames);
+            } else if (aFamilyNames != null) {
+                authors.add(aFamilyNames);
+            } else if (aAlias != null) {
+                authors.add(aAlias);
+            }
+        }
+
+        String authorStr = String.join(", ", authors);
+        entryMap.put(StandardField.AUTHOR, authorStr);
+
+        BibEntry entry = new BibEntry(StandardEntryType.Software);
+        entry.setField(entryMap);
+
+        List<BibEntry> entriesList = new ArrayList<BibEntry>();
+        entriesList.add(entry);
+
+        return new ParserResult(entriesList);
+    }
+
+    @Override
+    public boolean isRecognizedFormat(BufferedReader reader) throws IOException {
+
+        ObjectMapper mapper = new ObjectMapper(new YAMLFactory());
+        CffFormat citation;
+
+        try {
+            citation = mapper.readValue(reader, CffFormat.class);
+        } catch (IOException e) {
+            return false;
+        }
+
+        if (citation != null && citation.vals.get("title") != null) {
+            return true;
+        } else {
+            return false;
+        }
+    }
+
+    private HashMap<String, StandardField> getFieldMappings() {
+        HashMap<String, StandardField> hm = new HashMap<String, StandardField>();
+        hm.put("title", StandardField.TITLE);
+        hm.put("version", StandardField.VERSION);
+        hm.put("doi", StandardField.DOI);
+        hm.put("license", StandardField.LICENSE);
+        hm.put("repository", StandardField.REPOSITORY);
+        hm.put("url", StandardField.URL);
+        hm.put("abstract", StandardField.ABSTRACT);
+        hm.put("message", StandardField.COMMENT);
+        hm.put("date-released", StandardField.DATE);
+
```

Comment:
For fields that can't be mapped, you can simply create a new UnkownField: (example is taken from the MSBibImporter)
And set CFF_PREFIX to sth ilke `cff-`
`new UnknownField(CFF_PREFIX + "medium"), "Medium"); `

JabRef can show non-standard fields in the entry editor as well

Reply:
I think there should be a better solution than prefixing those fields, since none of the citation styles would pick them up - but you might want to have the repository printed in the reference list. I've started a discussion at the biblatex repo how to handle those fields: https://github.com/plk/biblatex/issues/1169